src/py/covidr/app.py

```diff
@@ -1,25 +1,104 @@
 import os
 from pathlib import Path
+from functools import wraps
+
+from dotenv import find_dotenv, load_dotenv
+from flask import Flask, send_from_directory, redirect, session, url_for
+from authlib.integrations.flask_client import OAuth
+from six.moves.urllib.parse import urlencode
```

Comment:
We don't support python 2, so we can just import urllib.parse's urlencode.

Reply:
good point